src/modules/FioAddress/util.js

```diff
@@ -618,7 +621,9 @@ const buyAddressRequest = async (
     })
 
     if (buyAddressResponse.error) {
-      console.log(buyAddressResponse.error)
+      if (buyAddressResponse.error === INVALID_DOMAIN_ERROR_1 || buyAddressResponse.error === INVALID_DOMAIN_ERROR_2) {
```

Comment:
How is this expected to work is the `buyAddressResponse.error` that we get from the Fio plugin is never equal to any of those strings?

Instead, the currencyInfo for Fio already exposed the possible error codes so you should use them to check what is the type of the error.

Here you can find the error codes:
https://github.com/EdgeApp/edge-currency-accountbased/blob/1724bf40a7d431fb2ee068da9138ef62723ae2ec/src/fio/fioInfo.js#L41